locale/en/blog/wg/diag-wg-update-2017-02.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Diag WG Update - Lots of new goodies, and phasing out some old ones
+author: Josh Gavant (@joshgav)
+date: 2017-02-28T12:00:00.000Z
+category: wg
+status: publish
+slug: diag-wg-update-2017-02
+layout: blog-post.hbs
+---
+
+In surveys and discussions with Node.js users we consistently hear of your need
+for better tools and APIs for debugging and understanding your modules and apps.
+In fact, in last year's [Node.js Foundation survey][], the biggest ask from
+developers and technical leads was for better tools.
+
+[Node.js Foundation survey]: https://nodejs.org/static/documents/2016-survey-report.pdf
+
+Based on that feedback and thanks to contributions and collaboration from across
+our community, over the past year several experimental diagnostic APIs and tools
+have landed in Node itself and the Node.js Foundation, including:
+
+* [async_hooks][] for sharing context across async boundaries,
+* [Inspector][] and [node-inspect][] for step debugging and profiling, 
+* [Trace Controller][] for gathering and streaming traces from V8 and Node, and
+* [node-report][] and [llnode][] for post-mortem analysis.
+
+[async_hooks]: https://github.com/nodejs/node/pull/8531
+[Inspector]: https://github.com/nodejs/node/issues?utf8=%E2%9C%93&q=label%3Ainspector%20
+[node-inspect]: https://github.com/nodejs/node-inspect
+[Trace Controller]: https://github.com/nodejs/node/pull/9304
+[node-report]: https://github.com/nodejs/node-report
+[llnode]: https://github.com/nodejs/llnode
+
+Over the coming months we hope to graduate these from experimental status with
+your help and feedback. Search the nodejs GitHub org for labels [diag-agenda][],
+[tracing][], and [inspector][] and review issues in the [Diagnostics WG][] to
+see what we're working on and how you can help.
+
+[diag-agenda]: https://github.com/search?q=org%3Anodejs+label%3A%22diag-agenda%22&type=Issues
+[tracing]: https://github.com/search?utf8=✓&q=org%3Anodejs+label%3A"tracing"+is%3Aopen&type=Issues
+[inspector]: https://github.com/search?utf8=✓&q=org%3Anodejs+label%3A"inspector"+is%3Aopen&type=Issues
+[Diagnostics WG]: https://github.com/nodejs/diagnostics/issues
+
+## Inspector replaces legacy V8 Debugger
+
+With progress comes the need to phase out old implementations so we can focus on
+refining and completing new ones to meet your needs. In particular, as Inspector
+[graduates from experimental
+status](https://github.com/nodejs/node/issues/11421) in the coming months, V8
+and Node's legacy Debugger API will be
+[deprecated](https://github.com/nodejs/node/pull/10970) and eventually
+[removed](https://github.com/nodejs/node/issues/9789) in favor of the new
+Inspector API.
+
+As a result, **running `node --debug`** (or its variants like `--debug-brk` and
+`--debug-port=1234`) **will present a deprecation warning now and stop working
+in a future release**. To debug your code from now on, use `node --inspect` or
```

Comment:
If the warning is going to be added just in v7 I think we should be specific on that